.env.example

```diff
@@ -0,0 +1 @@
+ARCHIVE_NODE_URL='your infura or alchemy url here'
```

Comment:
What are we requiring an ETH RPC for? Do we need historical ETH state?

Reply:
This is for testing only